frontend/src/components/elements/Favicon/Favicon.tsx

```diff
@@ -30,19 +45,6 @@ function overwriteFavicon(imageUrl: string): void {
   }
 }
 
-// An SVG for a full-sized emoji, encoded as a data URI
-function svgURI(emoji: string | null, scale: number): string {
-  return `data:image/svg+xml,
-  <svg version="1.2" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
-    <text
-      style="transform: translate(50%, 50%) scale(${scale})"
-      dominant-baseline="central"
-      text-anchor="middle">
-      ${emoji}
-    </text>
-  </svg>`
-}
-
 // Return the emoji if it exists, or empty string otherwise
 function extractEmoji(maybeEmoji: string): string {
```

Comment:
Not really part of this PR but assuming it doesn't fail I would put `nodeEmoji.hasEmoji(maybeEmoji)` as the first condition. I like to put the conditions that require the least work first